guides/assets/stylesrc/_main.scss

```diff
@@ -56,8 +64,9 @@ body.guide {
     hyphenate-limit-last: always;
 
     margin: 0 0 0.75em 0; // Space after paragraph
-    text-align: left;
 
+    :where(html[dir="ltr"]) & { text-align: left; }
+    :where(html[dir="rtl"]) & { text-align: right; }
```

Comment:
More of thinking out loud, to hear your thoughts: would it work / make sense to style left by default, and use the dir=rtl tag to swap right, rather than having the two explicitly called out throughout the styles? In other words, by default (without thinking about "dir") it is listed there / it works... when we swap is that the extra dir is taken into account for changing stuff.

I don't mind it this way, just something that crossed my mind.

Reply:
I don't have a strong opinion here, but the arguments for both methods:

Keeping as is:

1. Pro: Clarity of what parameters are being addressed and how they change
2. Con: If `dir` is not defined, it falls back to the cascade and works up the tree until it finds a declaration or the browser default.

Moving to only defining the override is effectively the opposite. I stuck with the method that was previously employed in the guides build, and I do like the visual clarity of the elements that change being spelled out next to each other, but it's not a hill I would die defending.